tasks/rh-sign-cosign-image/rh-sign-cosign-image.yaml

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: rh-sign-cosign-image
+  labels:
+    app.kubernetes.io/version: "1.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task to sign container images in a Snapshot spec using the cosign.
+  params:
+    - name: snapshotPath
+      description: Path to the JSON string of the mapped Snapshot spec in the data workspace
+      type: string
+      default: "snapshot_spec.json"
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
+      type: string
+      default: "data.json"
```

Comment:
Can you remove this default value? We changed all of our tasks some time ago to not have default values for the *Path parameters. We found it was leading to us missing some things in our pipeline definitions